src/application/agora/schema.ts

```diff
@@ -0,0 +1,47 @@
+import { z } from 'zod';
+
+export const proposalSchema = z.object({
+  proposalId: z.string(),
+  contractAddress: z.string(),
+  proposerAddress: z.string(),
+  description: z.string(),
+  ordinal: z.string(),
+  createBlock: z.string(),
+  startBlock: z.string(),
+  endBlock: z.string(),
+  cancelledBlock: z.string(),
+  executedBlock: z.string(),
+  proposalData: z.object({
+    votingStrategy: z.enum(['STANDARD']),
+    title: z.string(),
+    startTimestamp: z.string().datetime(),
+    endTimestamp: z.string().datetime(),
+    createdTimestamp: z.string().datetime(),
+    link: z.string(),
+    scores: z.string().array(),
+    votes: z.string(),
+    state: z.enum(['ACTIVE']),
+  }),
+  proposalTemplate: z.object({
+    name: z.string(),
+    proposalTemplateId: z.number(),
+    votingStrategy: z.enum(['STANDARD']),
+    quorum: z.string(),
+    approvalThreshold: z.string(),
+    contractAddress: z.string(),
+    createBlock: z.string(),
+  }),
+});
```

Comment:
I think we don't need most of this, let's cut down the schema to things we will use, if they change any field that we don't use we're still fine

Reply:
Good idea